ci/docker-compose.yml

```diff
@@ -45,7 +42,6 @@ services:
       CONJUR_APPLIANCE_URL: http://conjur
       CONJUR_ACCOUNT: cucumber
       DATABASE_URL: postgres://postgres@pg/postgres
-      AUDIT_DATABASE_URL:
```

Comment:
What is the affect if a client leaves this audit_datbase_url and give it value? is it removal reflected in code?

Reply:
We not longer used AUDIT_DATABASE_URL env variable since the audit table is part of the main db